CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ option(CSP_USE_LD_CLASSIC_MAC "On macOS, link with ld_classic" OFF)
 # Extension options
 option(CSP_BUILD_KAFKA_ADAPTER "Build kafka adapter" ON)
 option(CSP_BUILD_PARQUET_ADAPTER "Build parquet adapter" ON)
+option(CSP_BUILD_WS_CLIENT_ADAPTER "Build ws client adapter" ON)
```

Comment:
Please enable the corresponding flag in `setup.py` (it was merged yesterday, apologies)